### ISSUE TITLE
Static code review

### ISSUE THREAD

**Issue by szilagyinorbi** [szilagyinorbi/Habito#7]
Travis doesn't seem to work, try make static review instead on some files.

**Comment by Neo16** [comment]
Ok. Sorry I think I made a duplicate issue for this one.

**Comment by szilagyinorbi** [comment]
Ok. I reviewed them and they seemed fine.

**Comment by micskeiz** [comment]
It is a bit hard to follow your progress, as seemingly irrelevant issues are referenced.

For the code review it is not obvious what things you checked in the code, a checklist or a summary could be added to the wiki page.
